polaris_wyre/wyre/wyre.py

```diff
@@ -0,0 +1,35 @@
+from typing import Tuple
+
+from polaris_wyre.wyre.dtos import TransferData
+from .api import TEST_BASE_URL, WyreAPI
+
+COMPLETED_STATUS = "COMPLETED"
+FAILED_STATUS = "FAILED"
+
+
+# TODO: add methods description
+class Wyre:
+    def __init__(
+        self, api_token: str = "", account_id: str = "", api_url: str = TEST_BASE_URL
+    ):
+        self.wyre_api = WyreAPI(
+            api_token=api_token, account_id=account_id, api_url=api_url
+        )
+
+    def get_account(self) -> Tuple[str, str]:
+        response_data = self.wyre_api.get_account()
+        return response_data["depositAddresses"]["XLM"].split(":")
+
+    def get_stellar_transaction_id(self, transfer_id) -> str:
+        transfer_is_completed = False
+        while not transfer_is_completed:
```

Comment:
Maybe use `while True` and `break` instead of using the `transfer_is_completed` variable.